### ISSUE TITLE
Unqualified reference inspection ignores use statements

### ISSUE THREAD

**Issue by tobias-trozowski** [kalessil/phpinspectionsea#650]
Adding `use function <func>;` will have same result as prefixing the function call with `\`. See https://3v4l.org/FSu2v/vld#output

**Actual behaviour:**
The unqualified reference inspection ignores this and adds 2 warnings. One for the function call itself and one for the function name in the use statement.
```php
<?php
namespace Foo\Bar;
use function dirname;
dirname('foo/bar.baz');
```

**Expected behaviour:**
The inspection will not add any warning in this case.

**Comment by tobias-trozowski** [comment]
Just recognized that function names in use statement are treated as normal function calls.
`use function get_class;` gets a warning that the `__CLASS__` constant should be used instead.

**Comment by kalessil** [comment]
Thank you for reporting @tobias-trozowski. 

`use function get_class` is a good catch, I'll fix it. 

Importing functions via use statement support to be considered - I'm lacking time resources.

**Comment by Ocramius** [comment]
Is this fix in a released version? Just had the same issue, but I see a related commit and the issue still open. Just wondering btw.

**Comment by kalessil** [comment]
It's 2 cases here: respecting functions import is not fixed yet.

**Comment by kalessil** [comment]
Scheduled for the next release.

**Comment by kalessil** [comment]
Fixed
